static/app/utils/theme/theme.chonk.tsx

```diff
@@ -824,3 +823,116 @@ declare module '@emotion/react' {
     isChonk: boolean;
   }
 }
+
+/**
+ * Chonk utilities and overrrides to assert correct theme type
+ * inside chonk components without having to check for theme.isChonk everywhere
+ */
+// eslint-disable-next-line @typescript-eslint/naming-convention
+interface DO_NOT_USE_ChonkTheme extends ChonkTheme {
+  isChonk: true;
+}
+
+interface ChonkCreateStyled {
```

Comment:
Maybe add a comment here linking to the Emotion source? Knowing this is just a temporary fork + override will be helpful when we come back to pull this back out!

Reply:
Good call, I added it below but misses this one!